.github/workflows/iroha2-dev-pr.yml

```diff
@@ -44,7 +45,9 @@ jobs:
       - uses: actions/checkout@v4
       - uses: Swatinem/rust-cache@v2
       - name: Run tests, with coverage
-        run: mold --run cargo test --all-features --workspace --no-fail-fast
+        run: |
+          mold --run cargo test --all-features --no-fail-fast --workspace --exclude iroha_client
+          mold --run cargo test --all-features --no-fail-fast -p iroha_client -- --skip integration
```

Comment:
![image](https://github.com/hyperledger/iroha/assets/49983831/e4efede5-ec0e-47b5-81d0-94ed5adf89b9)

However, the result of 18% would not indicate that coverage has returned:

![image](https://github.com/hyperledger/iroha/assets/49983831/05ed1aae-bc79-47a2-b578-ea21edbb1c0a)
https://coveralls.io/builds/67405583

Something is wrong. There are some weird items like `target/` and `usr/`

Reply:
Tried several patterns to little effect. Reopened since at least `with_coverage` step works

Reply:
- Left to #4584